src/h5web/visualizations/containers/HeatmapVisContainer.tsx

```diff
@@ -11,10 +14,10 @@ import { VisContainerProps } from './models';
 function HeatmapVisContainer(props: VisContainerProps): ReactElement {
   const { entity, entityName } = props;
   assertDataset(entity);
+  assertSimpleShape(entity);
+  assertNumericType(entity);
```

Comment:
Adding assertions here for consistency with `supportsEntity`, and to avoid the type cast below.